### PR TITLE
Avoid matching sourceMappingURL inside a template string

### PR DESCRIPTION
**Pull request by aleclarson**
Proof for issue #60

### CODE REVIEW

**Comment by aleclarson** [comment]
Patch added. LGTM

File: index.js [hunk 1]
```diff
@@ -10,9 +10,7 @@ Object.defineProperty(exports, 'commentRegex', {
 
 Object.defineProperty(exports, 'mapFileCommentRegex', {
   get: function getMapFileCommentRegex () {
-    //Example (Extra space between slashes added to solve Safari bug. Exclude space in production):
-    //     / /# sourceMappingURL=foo.js.map           /*# sourceMappingURL=foo.js.map */
-    return /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
+    return /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"`]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
   }
```
Comment (thlorenz) [inline index.js]: Could we please keep the comments that were there?

**Comment by thlorenz** [comment]
LGTM after the nit is fixed. Would like at least one more collab to have a look before merging.

File: index.js (hunk 1 above)
Comment (aleclarson) [reply index.js]: How about something more like:
```js
// Matches sourceMappingURL in either // or /* comment styles.
```

The current comment interferes with global packages where two installations of `convert-source-map` end up parsing each other, causing an error to be thrown.

**Comment by thlorenz** [comment]
@aleclarson made you collaborator.

Please merge to master following [this guide](https://gist.github.com/thlorenz/f7df0daf13c6f6c1fa84).
I will then version and publish to npm.

Thanks.

**Comment by aleclarson** [comment]
@thlorenz Done 👍

**Comment by thlorenz** [comment]
Thanks published as patch `v1.5.1`.

**Pull request closed without merging**